web/app/js/components/Navigation.jsx

```diff
@@ -189,7 +189,7 @@ class NavigationBase extends React.Component {
     window.addEventListener("resize", this.updateWindowDimensions);
   }
 
-  componentWillUpdate() {
```

Comment:
This method is deprecated, they recommend: `Typically, this method can be replaced by componentDidUpdate(). If you were reading from the DOM in this method (e.g. to save a scroll position), you can move that logic to getSnapshotBeforeUpdate().`